### PR TITLE
Centralize scroll on both forms and forms systems applications

### PR DESCRIPTION
**Pull request by CBonade**
## Description
Centralize scroll on both forms and forms systems applications. This is a precursor to accessibility initiatives around scrolling.

## Original issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/6844

### CODE REVIEW

File: src/platform/forms-system/src/js/containers/FormPage.jsx
Comment (bkjohnson) [inline src/platform/forms-system/src/js/containers/FormPage.jsx:31]: Can you describe the differences between these two `scrollToTop` functions?

https://github.com/department-of-veterans-affairs/vets-website/blob/faf2613a8ac774708f7850b14d7b019ad3895985/src/platform/utilities/ui/scrollToTop.js#L3-L12

Also, this old version is passing in an object if `window.Forms?.scroll` doesn't exist, and its replacement is passing in an empty string instead. What is the impact of that change?

File: src/platform/forms-system/src/js/fields/ArrayField.jsx
Comment (bkjohnson) [inline src/platform/forms-system/src/js/fields/ArrayField.jsx:20]: Do we still need these?

File: src/platform/forms-system/src/js/containers/FormPage.jsx
Comment (CBonade) [reply src/platform/forms-system/src/js/containers/FormPage.jsx:31]: https://github.com/department-of-veterans-affairs/vets-website/pull/18770#discussion_r711345683 Going to link to this comment on the other PR as the explanation is the same I believe.

File: src/platform/forms-system/src/js/fields/ArrayField.jsx
Comment (CBonade) [reply src/platform/forms-system/src/js/fields/ArrayField.jsx:20]: the Element is still being referenced further down, however I can remove the scroller reference. Will push that up shortly.

File: src/platform/forms-system/src/js/containers/FormPage.jsx
Comment (CBonade) [reply src/platform/forms-system/src/js/containers/FormPage.jsx:31]: I missed the first part of your question above the code block - these were both intended to be converted and I totally overlooked the first one. I made that correction in my PR now to keep them both the same.

File: src/platform/forms-system/src/js/containers/FormPage.jsx
Comment (bkjohnson) [inline src/platform/forms-system/src/js/containers/FormPage.jsx:35]: Same concern here about the empty string.

File: src/platform/forms-system/src/js/review/ReviewPage.jsx
Comment (bkjohnson) [inline src/platform/forms-system/src/js/review/ReviewPage.jsx:16]: Here as well.

**Review by bkjohnson — COMMENTED** [review]
Like the other PR, I'd rather rely on `window.whatever?.scroll` being undefined as a way of triggering the `getScrollOptions()` default parameter instead of passing in an empty string.

File: src/platform/forms-system/src/js/review/ReviewPage.jsx
Comment (CBonade) [reply src/platform/forms-system/src/js/review/ReviewPage.jsx]: Updated

File: src/platform/forms-system/src/js/containers/FormPage.jsx
Comment (CBonade) [reply src/platform/forms-system/src/js/containers/FormPage.jsx]: Updated

File: src/platform/forms-system/src/js/containers/FormPage.jsx
Comment (bkjohnson) [inline src/platform/forms-system/src/js/containers/FormPage.jsx:24]: Same thing here: https://github.com/department-of-veterans-affairs/vets-website/pull/18770#discussion_r712523738

Comment (CBonade) [reply src/platform/forms-system/src/js/containers/FormPage.jsx:24]: > I'm going to hold making any adjustments to these. There was another request for additional refinement having to do with these and once that's merged, this will be handled by the `getScrollOptions` function. Any inline declarations like this will be replaced with that function so this will be a non-issue. 

Quoting my reply from another thread - putting this on hold til the other PR is merged that centralizes the `getScrollOptions` function and then will update this and test accordingly.

**Comment by CBonade** [comment]
@bkjohnson I've revised this in all areas to match the solution we arrived at in the other PR. This should keep a nice standard approach across all these files.

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**
